docs/HowTo/Get-Started/System-Requirements.md

```diff
@@ -63,12 +63,16 @@ How you synchronize with the Ethereum MainNet and whether you
 [Full synchronization](../../Reference/CLI/CLI-Syntax.md#sync-mode), which disables
 [pruning](../../Concepts/Pruning.md) by default, requires approximately 3TB.
 
-## VM requirements
+## AWS requirements 
+
+For mainnet nodes on AWS: 
 
-You can run Besu on a virtual machine (VM) on a cloud service, such as AWS or Azure, or locally
-using a VM manager, such as [VirtualBox](https://www.virtualbox.org/).
+* The minimum box size is t3.large.
+* The recommended box size is t3.xlarge. 
```

Comment:
Can optionally add: `This recommendation is to speed up the sync process by giving it more resources, after the sync is complete it can be dropped back to a t3.large`